db/data_migrations/20180101000000_initial_seed.rb

```diff
@@ -1,8 +1,8 @@
 class InitialSeed < ActiveRecord::DataMigration
   def up
-    require Rails.root("db/seeders/correspondence_type_seeder")
-    require Rails.root("db/seeders/case_closure_metadata_seeder")
-    require Rails.root("db/seeders/report_type_seeder")
+    require Rails.root.join("db/seeders/correspondence_type_seeder").to_s
```

Comment:
Looks good, although I don't think the `to_s` is required.

Reply:
Yes, `.to_s` attempts to convert the response to a string. It won't be needed here.